include/deal.II/base/partitioner.templates.h

```diff
@@ -329,6 +329,43 @@ namespace Utilities
         return a;
       }
 
+      // In the import_from_ghosted_array_finish we need to calculate maximal
```

Comment:
we might need

Reply:
The code was updated and rebased to avoid the build-error. 
Test coverage for the read write vector was added.
Test coverage for initialized and non-initizalized ghost values was added. Also, the issue of calling VectorOperation::min max twice subsequently was addressed in a comment before the compress function and additionally the test was extended to show the behavior in this case.